src/index.css

```diff
@@ -1,13 +1,41 @@
+@import url('styles/utils.css');
+
+h1, h3, p {
+    margin: 0;
+}
+
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
-  -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  box-sizing: border-box;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+#root {
+    height: 100vh;
+    overflow-y: scroll;
+    margin: 0 auto;
+    background-image: linear-gradient(rgb(255, 255, 255), rgb(212, 212, 212));
+    padding: 0 25%;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+
+@media screen and (min-width: 450px) and (max-width: 767px) {
+    #root {
+        padding: 0 5%;
+    }
 }
+
+@media screen and (max-width: 449px) {
+    #root {
+        padding: 0 5%;
+    }
+
+    .amount {
+        font-size: 1.5rem;
+    }
```

Comment:
This class name is used in two different files. Keep related CSS in a single file